sdk/keyvault/Azure.Security.KeyVault.Keys/src/KeyClient.cs

```diff
@@ -34,7 +34,7 @@ public KeyClient(Uri vaultUri, TokenCredential credential, KeyClientOptions opti
 
             _pipeline = HttpPipelineBuilder.Build(options,
                     bufferResponse: true,
-                    new BearerTokenAuthenticationPolicy(credential, "https://vault.azure.net//.default"));
+                    new BearerTokenAuthenticationPolicy(credential, "https://vault.azure.net/.default"));
```

Comment:
Does this value ever need to be configured by the user?  Especially for non-public clouds?  I don't think you need to make that change as part of your test improvements.

Reply:
Good point. It might change with challenged based auth which Scoot is working on